### DISCUSSION TITLE
Russian Language Localization

### DISCUSSION THREAD

**Discussion by Ksenia-from-OsoBorroso** [mozilla/hubs#3174] [General Questions & Discussion] [answered]
Hi there,

I've made a translation of the original English language file into Russian.

https://drive.google.com/file/d/11wwX4YTtsG9FxwsaofZ1f324iIoiOoct/view?usp=sharing

I noticed that the file lacks some important lines. Please contact me to translate them, too.

**Comment by johnshaughnessy** [comment, marked as answer]
Hi @Ksenia-from-OsoBorroso !

I was not able to access the file you linked so I sent a request to. 

@elmau (https://github.com/elmau) also mentioned wanting to add Russian translations. Are you working together or is this a coincidence?

**Reply by Ksenia-from-OsoBorroso** [reply]
Yeah, we are working together, it's the same translation.

I gave you access. Sorry for making it a bit complicated.

**Reply by elmau** [reply]
Ksenia is the one doing the translations, I submit them  :)

**Reply by elmau** [reply]
@johnshaughnessy were you able to access the file?

**Reply by johnshaughnessy** [reply]
@elmau Yes, I received the translation file via email. I haven't had a chance to create a pull request to add them yet but it's on my todo list

**Reply by elmau** [reply]
@johnshaughnessy I think @keianhzo also has the file.

**Reply by robinkwilson** [reply]
Marking as answered, click `Unmark as answered` button if not resolved!

**Comment by johnshaughnessy** [comment]
@Ksenia-from-OsoBorrosso @elmau - I've (finally) created a pull request to add the Russian localization. There are a few lines that are missing : https://github.com/mozilla/hubs/pull/3336/commits/62c0ff79aa8dbd23aad7f43fe572721da74c9608#diff-fd3ceec3f33816f68d73b35a3438016797891aa83e430883f206c09244dba83aR243-R248
If you send the translations for those lines, I'll add them.

Thanks again for your contributions! I'll get this reviewed and deployed early next week.

**Reply by Ksenia-from-OsoBorroso** [reply]
@johnshaughnessy sorry for not answering for long. No problem, I'll translate them next week. Take care!

**Reply by Ksenia-from-OsoBorroso** [reply]
Hi @johnshaughnessy, I've just finished the additional translation you had asked me for.

Please let me know if any other line is missed. I have no problem at all to keep perfecting the Russian and Spanish versions.
